tactics/class_tactics.ml

```diff
@@ -643,8 +654,11 @@ module Search = struct
            (if backtrack then str" with backtracking"
             else str" without backtracking"));
     let secvars = compute_secvars gl in
-    let poss =
-      e_possible_resolve hints info.search_hints secvars info.search_only_classes env sigma concl in
+    match e_possible_resolve hints info.search_hints secvars
+            info.search_only_classes env sigma concl with
+    | ModeMismatch ->
+      Proofview.tclZERO ~info:Exninfo.null StuckClass
```

Comment:
Why `null`? cc @ejgallego

Reply:
Indeed better leave it unassigned, if we merge #11755 things should work.

Reply:
This needs to be fixed before merge.

Reply:
Fixed